news/2019-02-04-community-choice-2018-results.md

```diff
@@ -0,0 +1,147 @@
+---
+layout: post
+title: "Community Choice 2018: Results"
+date: 2019-02-04 10:30:00 +0000
+---
+
+You've had your say, and we have our winners! Check out the new Community Favourites of 2018, featuring some of the closest voting we've ever seen!
+
+[![](/wiki/shared/news/2019-01-17-community-choice-2018/header.jpg)](https://osu.ppy.sh/community/contests/74)
+
+### osu!
+
+[![](/wiki/shared/news/2019-02-04-community-choice-2018-results/kda.jpg)](https://osu.ppy.sh/beatmapsets/874948)
+
+**2489** voters could feel the rush now as they enjoyed [Nathan](https://osu.ppy.sh/users/4785223)'s take on the video-game cross K-pop masterpiece that took the world by storm, **[K/DA - POP/STARS (ft. Madison Beer, (G)I-DLE, Jaira Burns)](https://osu.ppy.sh/beatmapsets/874948)**!
+
+This will likely come as a surprise to some who expected one map from the ubiquitous deluge of "farm" maps to nab top place, but as this goes to show, sometimes you just need to stop farming creeps and get out there and start nabbing kills, something that Nathan and his collaborators managed with effortless ease.
```

Comment:
Is it intentional to repeat "farm" and "nab" here? Variety would be nice. Also I get that you were linking in with league r.e. the "kills" comment, but it makes no sense in this context because you're comparing it to votes... Do people die when they vote??